question-service/README.md

```diff
@@ -18,6 +18,15 @@
   ```
 4. FYI: I used the nodemon dependency so that you can save your code changes and the backend server will automatically restart (observe your terminal for more information)
 
+### How to run the containerized backend server with docker:
+ - 1. Install docker on your device: [Docker](https://www.docker.com)
+ - 2. Setup the `.env` file as the above paragraph mentioned
+ - 3. Run `yarn run docker-build-image` to build the image (named `question-service-image`)
+ - 4. Run `yarn run docker-create-container` to create a default container (named `question-service`). This will create and run the default container. Now the server should be accessible via the port `3000`.
+ - 5. Run `yarn run docker-start` to (re)start the already-created default container. 
+ - 6. Run `yarn run docker-stop` to stop the default container.
```

Comment:
since the question service is currently using `npm`, might want to change the commands here to use `npm run` instead

Reply:
so just replace all instances of `yarn run` with `npm run`